### ISSUE TITLE
Kia India

### ISSUE THREAD

**Issue by Gopithub** [Hyundai-Kia-Connect/kia_uvo#650]
AS suggested, raising an issue for India. BTW, What should be the repository name I should give in Home Assistant and where am I supposed to give it? The path that is mentioned in the documentation, I could not find an option to add a repository. But I could find another place where adding repositories are available and that is where I tried it.

I am using Rasp pi special OS for Home Assistant

**Comment by cdnninja** [comment]
Someone needs to sniff the traffic and work to reverse engineer it.  

It needs to be added via HACS.

**Comment by Gopithub** [comment]
Not sure what that means:-) Anything required from my end?

**Comment by cdnninja** [comment]
Yes.  I need you to capture the traffic between the website or app and the server.

**Comment by Gopithub** [comment]
How to do?

**Comment by cdnninja** [comment]
It is outlined in the wiki.
